modules/microshift-building-apps-rpms.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-apps-on-rhel-edge.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-building-apps-rpms_{context}"]
+= Building the RPM package for the application manifests
+
+To building your own RPMs, you must create a `.spec` file that adds the application manifests to the RPM package. The following is an example procedure. As long as your RPMs are accessible to Image Builder, you can use the method you prefer.
+
+.Prerequisites
+* You have set up a Red Hat Enterprise Linux (RHEL) for Edge 9.2 build host that meets the Image Builder system requirements.
+* You have root access to the host.
+* The file tree required to build RPM packages was created.
+
+.Procedure
+
+. In the `~/rpmbuild/SPECS` directory, create a file named `<application-workload-manifests.spec>` using the following example content:
++
+.Example `.spec` file
++
+[source,terminal]
+----
+Name: application-workload-manifests
+Version: 0.0.1
+Release: 1%{?dist}
+Summary: Adds workload manifests to microshift
+BuildArch: noarch
+License: GPL
+Source0: %{name}-%{version}.tar.gz
+#Requires: microshift
+%description
+Adds workload manifests to microshift
+%prep
+%autosetup
+%install <1>
+rm -rf $RPM_BUILD_ROOT
+mkdir -p $RPM_BUILD_ROOT/%{_prefix}/lib/microshift/manifests
+cp -pr ~/manifests $RPM_BUILD_ROOT/%{_prefix}/lib/microshift/
+%clean
+rm -rf $RPM_BUILD_ROOT
+
+%files
+%{_prefix}/lib/microshift/manifests/**
+%changelog
+* <DDD MM DD YYYY username@domain - V major.minor.patch>
+- <your-change-log-comment>
+----
+[.small]
+<1> The `%install` section creates the target directory inside the rpm, `/usr/lib/microshift/manifests/`
+and copies the manifests from the source home directory `~/manifests`.
++
+[IMPORTANT]
+====
+All the required `.yaml` files must be in the  `/usr/lib/microshift/manifests/` directory, including a `kustomize.yaml` if you are using kustomize.
+====
+
+. Build your RPM package in the `~/rpmbuild/RPMS` directory by running the following command:
++
+[source,terminal]
+----
+$ sudo rpmbuild -bb ~/rpmbuild/SPECS/microshift-workload-manifests.spec
+----
+
+.Verification
+
+. Check for `.spec` file errors by running the following command:
++
+[source,terminal]
+----
+$ sudo rpmlint ~/rpmbuild/SPECS/<application-workload-manifests>.spec
```

Comment:
Couple of issues here:

- that should not run as root, so no sudo required.

- usually, one would run rpmlint *BEFORE* the rpmbuild, to find issues in the spec file. So this step here is not a good Verification step. 

- Also the rpmlint command does create some errors, as commented previously by John. 

So I suggest we simply remove this Verification step at all, or change it from rpmlint to a simply "lets looks in the output directory with
e.g.
```
$ tree ~/rpmbuild/RPMS
/home/builder/rpmbuild/RPMS
├── noarch
│   ├── ushift-workload-manifests-0.0.1-1.el9.noarch.rpm

```

Reply:
If it isn't helpful for the user, I'll remove it. We don't want to add details that will just be confusing.